ghost/core/core/server/api/endpoints/themes.js

```diff
@@ -15,6 +16,18 @@ module.exports = {
         }
     },
 
+    active: {
+        headers: {
+            cacheInvalidate: true
```

Comment:
I'm confused, why would we want to invalidate cache on a GET request 🤔

Reply:
My 🅱️